### DISCUSSION TITLE
Livewire click column

### DISCUSSION THREAD

**Discussion by alejandrotrevi** [rappasoft/laravel-livewire-tables#1321] [General]
Would something like this be useful to anyone? I could submit a pr 🤔 

```php
        // ...
        LivewireClickColumn::make('Action')
                ->title(fn ($row) => 'Delete')
                ->action('delete'),
        ];
    }

    public function edit(Client $client): void
    {
        // Delete the client
    }
```

**Comment by lrljoe** [comment]
What's the benefit of that vs setting attributes on a Button Column, or just using a view?

Advantages of just using a view on a column includes adding confirmation modals etc where relevant.

Always happy to review PRs etc, just not sure what the benefit would be here, and don't want you to spend hours writing the tests and docs.
If there's a good use case then happy to review it tho!
